CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
   * Make `PopupBridgeClient#POPUP_BRIDGE_NAME` package-private
   * Make `PopupBridgeClient#POPUP_BRIDGE_URL_HOST` package-private
   * Remove `PopupBridgeLifecycleObserver`
+* Add JS interface for Venmo app installed check
```

Comment:
Not sure if we want to match the changelog entries here: https://github.com/braintree/popup-bridge-ios/blob/main/CHANGELOG.md#300-2025-04-01